servicemesh-install/topics/appendix-threescale-adapter.adoc

```diff
@@ -92,7 +142,7 @@ oc patch deployment "${DEPLOYMENT}" --patch ''"${patch}"''
 ----
 
 == Routing service traffic through the adapter
-To drive traffic for your service through the 3scale adapter, you need to match the rule `destination.labels["service-mesh.3scale.net"] == "true"` that you previously created in the configuration, in the kind: `rule resource`.
+To drive traffic for your service through the 3scale adapter, you need to match the rule `destination.labels["service-mesh.3scale.net"] == "true"` that you previously created in the configuration, in the `kind: rule resource`.
```

Comment:
I think this is likely to be `kind: rule` as an example, see: https://istio.io/docs/concepts/policies-and-telemetry/#rules

Reply:
Correct, that's the format upstream. Changing to `kind: rule` resource. should suffice.

Reply:
@PhilipGough I removed "resource" from the tickmarks.